### PR TITLE
Character setup UI Tweaks

### PR DESCRIPTION
**Pull request by Mazianni**
## Description of changes

- Reorders some files relating to background stuff (order is now species-background-languages).
- Moves Species selection to the Background tab and alters it to work like cultures do.

## Why and what will this PR improve

UI is important. 

## Authorship

Andromeda-K22/Mazian

## Changelog

:cl:
tweak: updated the UI on species selection
tweak: species selection is now in the background tab.
/:cl:

### CODE REVIEW

File: code/modules/client/preference_setup/background/03_language.dm [hunk 1]
```diff
@@ -129,8 +131,11 @@
 			var/decl/language/lang_instance = GET_DECL(lang)
 			if(free_languages[lang])
 				LAZYADD(., "- [lang_instance.name] (required).<br>")
+				LAZYADD(., "<i><b>[lang_instance.desc ? lang_instance.desc : "No information avaliable."]</b></i></br>")
```
Comment (MistakeNot4892) [inline code/modules/client/preference_setup/background/03_language.dm]: Could just do `lang_instance.desc || "No information avaliable."`

File: code/modules/client/preference_setup/background/01_species.dm [hunk 2]
```diff
@@ -0,0 +1,88 @@
+/datum/category_item/player_setup_item/background/species
+	name = "Species"
+	sort_order = 1
+	var/hide_species = TRUE
+
+/datum/category_item/player_setup_item/background/species/save_character(datum/pref_record_writer/W)
+	W.write("species",                pref.species)
```
Comment (MistakeNot4892) [inline code/modules/client/preference_setup/background/01_species.dm]: Sorry, minor thing, but could you trim the odd spacing here and below?

**Comment by MistakeNot4892** [comment]
My only concern with the reorder is that you'll now need to back a tab to update your appearance/markings, which might not be ideal. Not sure how to fix this.

**Review by MistakeNot4892 — APPROVED** [review]

**Comment by AlexMorgan3817** [comment]
Screenshots? No?

**Comment by MistakeNot4892** [comment]
Screenshots got posted on Discord, looks fine. Just moves species over to the backgrounds tab like the culture selectors.

**Comment by AlexMorgan3817** [comment]
> Screenshots got posted on Discord, looks fine. Just moves species over to the backgrounds tab like the culture selectors.

They should be posted on PR, not in discord where they can be lost

**Comment by Mazianni** [comment]
I typically do all my development on the main discord. You should be in there.
https://cdn.discordapp.com/attachments/678826323728400394/883279491215020072/unknown.png
https://cdn.discordapp.com/attachments/678826323728400394/883319745154269264/unknown.png

**Comment by AlexMorgan3817** [comment]
> I typically do all my development on the main discord. You should be in there.

Bro, all important info should be on PR, not on external resources, i understand that you posted it on discord, but PR should have screenshots of what design you create to be comfortable reviewed without launching your PR on local or searching images on discord

**Comment by MistakeNot4892** [comment]
I've hidden the conversation. Contribs are not beholden to random people demanding screenshots particularly not from people who aren't reviewers.

**Pull request merged**